opt/scopeserver/dataserver/modules/gserver/s_pb2_grpc.py

```diff
@@ -1,454 +1,454 @@
 # Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
-# pylint: skip-file
```

Comment:
Same here as on s_pb2

Reply:
Merged master into this branch ;)

Reply:
Weird, still had to still correct some files...